<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-7EUDCAL" failures="3" tests="3" name="Test" time="0.064" errors="0" timestamp="2024-12-05T21:52:23 GMT-04:00">
  <testcase classname="Retry.SauceTests" name="logintest" time="0.011">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at Retry.SauceTests.logintest(SauceTests.java:13)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- logintest -->
  <testcase classname="Retry.SauceTests" name="logouttest" time="0.002">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at Retry.SauceTests.logouttest(SauceTests.java:27)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- logouttest -->
  <testcase classname="Retry.SauceTests" name="ordertest" time="0.001">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at Retry.SauceTests.ordertest(SauceTests.java:19)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- ordertest -->
</testsuite> <!-- Test -->
